### PR TITLE
Cfg: Adds experimental scope grafana.ini settings

### PR DESCRIPTION
**Pull request by bergquist**
This PR adds experimental settings for scope filters. 

Add the following to your custom.ini to configure the values.  

```
[scopes]
list_scopes_endpoint = "url"
list_dashboards_endpoint = "url"
```

### CODE REVIEW

File: pkg/services/featuremgmt/registry.go [hunk 1]
```diff
@@ -1223,6 +1223,17 @@ var (
 			RequiresRestart: true,
 			Expression:      "true", // enabled by default
 		},
+		{
+			Name:              "scopeFilters",
+			Description:       "enables scope filters",
```
Comment (bfmatei) [inline pkg/services/featuremgmt/registry.go:1228]:
```suggestion
			Description:       "Enables the use of scope filters in Grafana",
```

File: pkg/services/featuremgmt/toggles_gen.go [hunk 2]
```diff
@@ -662,4 +662,8 @@ const (
 	// FlagAlertingUpgradeDryrunOnStart
 	// When activated in legacy alerting mode, this initiates a dry-run of the Unified Alerting upgrade during each startup. It logs any issues detected without implementing any actual changes.
 	FlagAlertingUpgradeDryrunOnStart = "alertingUpgradeDryrunOnStart"
+
+	// FlagScopeFilters
+	// enables scope filters
```
Comment (bfmatei) [inline pkg/services/featuremgmt/toggles_gen.go:667]:
```suggestion
	// Enables the use of scope filters in Grafana
```

File: pkg/services/featuremgmt/toggles_gen.json [hunk 3]
```diff
@@ -2123,6 +2123,20 @@
...
       }
+    },
+    {
+      "metadata": {
+        "name": "scopeFilters",
+        "resourceVersion": "1708524225414",
+        "creationTimestamp": "2024-02-21T14:03:45Z"
+      },
+      "spec": {
+        "description": "enables scope filters",
```
Comment (bfmatei) [inline pkg/services/featuremgmt/toggles_gen.json:2134]:
```suggestion
        "description": "Enables the use of scope filters in Grafana",
```

**Review by bfmatei — APPROVED** [review]
I would suggest having a more "formatted" description of the feature toggle. Other than that, LGTM 🚀 !

**Comment by bergquist** [comment]
@bfmatei If you need these changes before next week you can apply the changes, rebase and merge it :)

**Pull request merged**